.azure-documentation-release.yml

```diff
@@ -0,0 +1,92 @@
+resources:
+    repositories:
+        - repository: DocuTheme
+          type: github
+          endpoint: Innoactive
+          name: Innoactive/Hub-Documentation-DocFx-Theme
+
+trigger:
+  branches:
+    include:
+      - master
+      - develop
+  paths:
+    include:
+      - '.Documentation/*'
+
+pr: none
+
+parameters:
+- name: release
+  type: boolean
+  default: false
+
+pool: Unity
+
+jobs:
+  - job:
+    workspace:
+        clean: all
+
+    steps:
+      - checkout: DocuTheme
+        lfs: true
+      - checkout: self
+        lfs: true
+
+      - powershell: |
+          $TAG = (git describe --all --match "v*").Split("-")[0].Replace("tags/", "").TrimStart("v")
+          Write-Host "##vso[task.setvariable variable=Version]$TAG"
+        workingDirectory: 'Creator'
+        displayName: 'Parse build Tag'
+
+      - task: CopyFiles@2
+        inputs:
+          SourceFolder: 'Hub-Documentation-DocFx-Theme/'
+          Contents: '**'
+          TargetFolder: 'Creator/.Documentation/theme'
+        displayName: Copy Theme
+
+      - task: Npm@1
+        inputs:
+          command: 'custom'
+          workingDir: 'Creator/.Documentation/theme'
+          customCommand: 'install'
+        displayName: Prepare Theme
+
+      - task: Npm@1
+        inputs:
+          command: 'custom'
+          workingDir: 'Creator/.Documentation/theme'
+          customCommand: 'run build'
+        displayName: Build Theme
+
+      - task: NuGetCommand@2
+        inputs:
+          command: 'custom'
+          arguments: 'install docfx.console -Version 2.48.1'
+        displayName: Installing DocFx
+
+      - script: docfx.console.2.48.1\tools\docfx.exe build --globalMetadata "{_gaTrackingId:\"$(gaTrackingId)\",_version:\"v$(Version)\"}" Creator\.Documentation\docfx.json
+        displayName: "Building Documentation"
+
+      - task: PublishPipelineArtifact@1
+        inputs:
+          targetPath: 'Creator/.Documentation/_site'
+          artifact: 'documentation'
+          publishLocation: 'pipeline'
+        displayName: Store Documentation
+
+      - task: FtpUpload@2
+        displayName: "Uploading to server"
```

Comment:
Only step that could be considered "release" maybe it makes sence to also call it like that. Ideally move the release part to a separate job/stage